_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1339,19 +1338,22 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
 {
     m_core = core;
 
+    m_width  = width;
+    m_height = height;
+
+    memset(&m_caps, 0, sizeof(m_caps));
```

Comment:
where these 3 lines came from and why? they look suspicious. Maybe they are required, maybe not... Looks to be a separate bug fix. @stasefremov : can you, please, comment?

Reply:
These rows are came from the 1350 row. I set m_caps.FrameToleranceMode, so initializing of m_caps should be before it, not after.

Reply:
I managed to miss this, thank you.